_data/projects/Dive_into_Machine_Learning.yml

```diff
@@ -0,0 +1,12 @@
+name: Dive into Machine Learning
+desc: Dive into Machine Learning with Python Jupyter notebook and scikit-learn
+site: https://github.com/hangtwenty/dive-into-machine-learning
+tags:
+- machine-learning
+- ML
+- python
+- scikit-learn
+- data-science
+upforgrabs:
+  name: help%20wanted
```

Comment:
@shiftkey 

Oh, haha, good catch ... I wasn't sure if it need to exactly match, this makes sense though, updated to `help-wanted`